junit5-engine/src/main/java/org/junit/gen5/engine/junit5/descriptor/JUnit5TestableFactory.java

```diff
@@ -155,7 +155,11 @@ private Method findMethod(Class<?> clazz, String methodName, Class<?>[] paramete
 	}
 
 	private Class<?> loadClassByName(String className) {
-		return ReflectionUtils.loadClass(className).orElse(null);
+		return ReflectionUtils.loadClass(className).orElseThrow(() -> createCannotLoadClassException(className));
+	}
```

Comment:
Yes. There was a test that was expecting `IllegalArguementExxeption` that broke because we tripped on a `notNull()` further down the code.

This is another example of why the difference matters.